caml.py

```diff
@@ -0,0 +1,46 @@
+import numpy as np
+import pandas as pd
+import argparse
+import yaml
+
+# caml modules
+import clean
+
+""" This module is meant for control sequence """
+
+def read_input(input_file):
+    
```

Comment:
doc string goes below def?

Reply:
you can have a docstring for a module, but it should be the very first line, I think